tutorials/synthetic_data/01-synthetic-data-electronics.ipynb

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    from essential_generators import DocumentGenerator
</pre><p>You could have a cell to install those if you want to pin the version</p>

<pre class="ql-syntax" spellcheck="false">!python3 -m pip install awswrangler==3.7.2
!python3 -m pip install ssential_generators==1.0
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/python-deequ/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Will add a cell to demonstrate version pinning.